webpack.config.babel.js

```diff
@@ -189,6 +189,13 @@ const config = {
       index: 'index.html',
     },
   },
+
+  resolve: {
+    alias: {
+      'backpack-android': path.resolve(__dirname, 'backpack-android'),
+      'backpack-ios': path.resolve(__dirname, 'backpack-ios'),
```

Comment:
To be clear, this change allows us to do the following yeah?
```
import nativeIosReadme from 'backpack-ios/Backpack/Panel/README.md';
```

Reply:
Yeah, but I reverted it because of the eslint problem.